solr/core/src/java/org/apache/solr/cli/SimplePostTool.java

```diff
@@ -231,19 +235,13 @@ protected static SimplePostTool parseArgsAndInit(String[] args) {
 
       String params = System.getProperty("params", "");
 
-      String host = System.getProperty("host", DEFAULT_POST_HOST);
-      String port = System.getProperty("port", DEFAULT_POST_PORT);
-      String core = System.getProperty("c");
```

Comment:
If this is scheduled for 9.x it would be a pity to make the cli args of SimplePostTool non-back compatible by removing the "-c", "-host", "-port" options?

Reply:
I will have another think about this….   Maybe deprecation is more possible….

Reply:
Okay, I don't love it, but I restored it!  I will add a deprecation tag, since this whole class will be going away...

Reply:
Okay, done...   Any suggestions on how to mark things deprecated?  I did put in a message in bin/post...

Reply:
That is probably ok, the main thing is that in 10.0 when we do breaking change, we document the change in UPGRADE NOTES..